bcgov/nr-compliance-enforcement
# fix: Ce 478

Resets the values for the long and lat when the user presses the 'cancel changes' button.



# Description

Please provide a summary of the change and the issue fixed. Please include relevant context. List dependency changes.
The current code sets an interim value when the users changes the lat or long and then updates the map with that, when cancelling these values are retained when they should be reset to the originally saved values. 

Test procedure:
- Press 'Edit' on a complaint which as a lat and long
- When the modal comes up edit/remove the values in lat and long, confirm the map shows a different location
- Press the 'cancel changes' button and view the complaint to confirm it now has the original values and that the map it showing the previous/original location.
- Press 'Edit' again to confirm the original values are shown in the Modal.

Fixes # (issue)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration



## Checklist



## Further comments




---

Thanks for the PR!

Deployments, as required, will be available below:
- [Frontend](https://nr-compliance-enforcement-685-frontend.apps.silver.devops.gov.bc.ca)
- [Backend](https://nr-compliance-enforcement-685-frontend.apps.silver.devops.gov.bc.ca/api)


Please create PRs in draft mode.  Mark as ready to enable:
- [Analysis Workflow](https://github.com/bcgov/nr-compliance-enforcement/actions/workflows/analysis.yml)

After merge, new images are deployed in:
- [Merge Workflow](https://github.com/bcgov/nr-compliance-enforcement/actions/workflows/merge.yml)
